showcase/src/app/app.module.ts

```diff
@@ -120,7 +122,7 @@ export const metaReducers: MetaReducer<State>[] = ENV !== "production" ? [logger
  */
 @NgModule({
 	bootstrap: [AppComponent],
-	declarations: [AppComponent, HomeComponent, NoContentComponent, ActionBarComponent, ExampleViewerComponent],
+	declarations: [AppComponent, HomeComponent, NoContentComponent, NewsComponent, ActionBarComponent, ExampleViewerComponent, NewsItemComponent],
```

Comment:
Lets re-structure this "News" feature according to the Angular guidelines: all the functionalities in the app should be grouped by "feature" modules.

Then, the NewsComponent and the NewsItemComponent should be included in a NewsModule.

Then the NewsModule should be included in the "imports" of the AppModule.